src/server/index.ts

```diff
@@ -33,6 +37,17 @@ function readyLog(client: Client): void {
 
 client.login(token).catch(console.error);
 
-client.once('ready', () => {
+client.once('ready', async () => {
   readyLog(client);
+  await client.channels.fetch(mainChannelId).then((mainChannel) => {
+    if (!mainChannel || !mainChannel.isText()) {
+      throw new Error('メインのチャンネルが見つかりません。');
+    }
+    const provider = new VoiceRoomProxy<VoiceChannelParticipant>(
+      client,
+      (voicestate) => new DiscordParticipant(voicestate, mainChannel)
+    );
+    const runner = new VoiceRoomResponseRunner(provider);
+    runner.addResponder(new VoiceDiff());
+  });
```

Comment:
```suggestion
  const mainChannel = await client.channels.fetch(mainChannelId);
  if (!mainChannel || !mainChannel.isText()) {
    throw new Error('メインのチャンネルが見つかりません。');
  }
  const provider = new VoiceRoomProxy<VoiceChannelParticipant>(
    client,
    (voicestate) => new DiscordParticipant(voicestate, mainChannel)
  );
  const runner = new VoiceRoomResponseRunner(provider);
  runner.addResponder(new VoiceDiff());
```
`async` 使うなら `await` にしましょう